packages/adapter-node/src/server.js

```diff
@@ -57,9 +56,9 @@ export function createServer({ render }) {
 			const rendered = await render({
 				method: req.method,
 				headers: req.headers, // TODO: what about repeated headers, i.e. string[]
-				path: parsed.pathname,
-				query: parsed.searchParams,
-				rawBody: body
+				path: req.path,
```

Comment:
It's not clear to me that this PR is actually changing anything

I just created an `index.js` with the source code:
`console.log(new URL('http://localhost/über-uns').pathname)`

It printed:
>/%C3%BCber-uns

If I change the code to `console.log(new URL('http://localhost/%C3%BCber-uns').pathname)` it still prints the same thing

So it's actually not clear to me that this PR is changing anything. Am I missing something? Or can you explain what was happening before vs what's happening now?

Reply:
Well at the moment the render function is receiving, as the `path` value, the encoded string `/%C3%BCber-uns`.

And now after changing it to take the original path from the `req` object without transforming and encoding it with  URL class, the value that it receives is `/über-uns`

I think that the thing you're missing is, that my change for the adapter node, does not wrap the path in a url object, that encodes the string.